devicetypes/smartthings/zigbee-multi-switch.src/zigbee-multi-switch.groovy

```diff
@@ -105,22 +114,24 @@ private getChildEndpoint(String dni) {
 
 def on() {
 	log.debug("on")
-	zigbee.on()
+	zigbee.on() + zigbee.onOffRefresh()
 }
 
 def off() {
 	log.debug("off")
-	zigbee.off()
+	zigbee.off() + zigbee.onOffRefresh()
```

Comment:
Is there a reason you're having to add polls after all of your sets? That seems to indicate that reporting is not configured correctly.